internal/storage/user.go

```diff
@@ -0,0 +1,66 @@
+package storage
+
+import (
+	"context"
+	"github.com/google/uuid"
+	"github.com/poggerr/gophermart/internal/logger"
+	"github.com/poggerr/gophermart/internal/models"
+	"time"
+)
+
+func (strg *Storage) CreateUser(username, pass string, id *uuid.UUID) error {
+	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+	defer cancel()
+
+	_, err := strg.db.ExecContext(
+		ctx,
+		"INSERT INTO main_user (id, username, password, withdrawn, balance) VALUES ($1, $2, $3, $4, $5)",
+		id, username, pass, 0, 0)
+	if err != nil {
+		logger.Initialize().Info(err)
+	}
+	return nil
+}
+
+func (strg *Storage) VerifyUser(username string) bool {
```

Comment:
что значит verify? Здесь только проверка что username существует
```suggestion
func (strg *Storage) Exists(username string) bool {
```